API/Hardware_API.md

```diff
@@ -184,24 +184,93 @@ low\*     | pin voltage goes low  | Only available via pin.once()
 
 ### PWM pins
 
-**PWM pins** are pulse-width modulated pins ([wiki link](http://en.wikipedia.org/wiki/Pulse-width_modulation)). Essentially, PWM is a digital signal that spends between 0% and 100% of its time pulled high/on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
+**PWM pins** are pulse-width modulated pins. Essentially, PWM is a digital signal that spends between 0% and 100% of its time pulled high/on (this is its "duty cycle"). You can set the PWM pins to any value between 0 and 1 to approximate an analog signal. PWM is often used to control servo speeds or LED brightness.
```

Comment:
A little more detail on the applications would be useful. E.g. "LED brightness can be controlled this way because it is blinking so fast you don't see the flashes but just mentally average to a dimmer-than-fully-bright light"